src/main/java/com/untamedears/JukeAlert/command/commands/InfoCommand.java

```diff
@@ -52,17 +83,88 @@ public boolean execute(CommandSender sender, String[] args) {
             }
             final int snitchId = snitch.getId();
             int offset = 1;
+            LoggedAction filterAction = null;
+            String filterPlayer = "";
+            boolean censorFlag = false;
+            boolean nextFlag = false;
+            
             if (args.length > 0) {
-                try {
-                    offset = Integer.parseInt(args[0]);
-                } catch (NumberFormatException e) {
-                    if (playerPage_.containsKey(accountId)) {
-                        final History hist = playerPage_.get(accountId);
-                        if (hist != null && hist.snitchId == snitchId) {
-                            offset = hist.page + 1;
-                        } else {
-                            offset = 1;
+                
+                // Reassemble any arguments that are enclosed in quotes and were split
```

Comment:
Don't re-invent string parsing if you can avoid it.

See if https://docs.oracle.com/javase/7/docs/api/java/util/Scanner.html Scanner can do what you want; if not, consider a RegEx with group matches.